concent_api/gatekeeper/views.py

```diff
@@ -191,3 +201,15 @@ def parse_headers(request: WSGIRequest, path_to_file: str) -> Union[FileTransfer
     else:
         assert len(matching_files) == 0
         return gatekeeper_access_denied_response('Path to specified file is not listed in files variable.', path_to_file, loaded_golem_message.subtask_id, client_public_key)
+
+
+def is_request_from_concent_api(request) -> bool:
+    """
+    Checks if given request is sent from Concent API by checking HTTP_CONCENT_CLIENT_PUBLIC_KEY header.
+    """
+    if (
```

Comment:
it could be simplified to
```
return 'HTTP_CONCENT_CLIENT_PUBLIC_KEY' in request.META and
    decode_key(request.META['HTTP_CONCENT_CLIENT_PUBLIC_KEY']) == settings.CONCENT_PUBLIC_KEY
```

Reply:
Off course, but in our coding style we have:

`Eksplicite zwracamy wartość z funkcji za pomocą return, chyba że funkcja z założenia nie ma zwracać wartości (tzn. wszystkie ścieżki kończą się zwróceniem None).`

But I am not 100% sure if this is the case. @cameel can you remind me how this should be ?

Reply:
:+1: